src/components/transactions/Withdraw/WithdrawModalContent.tsx

```diff
@@ -183,6 +183,7 @@ export const WithdrawModalContent = ({
         isMaxSelected={isMaxSelected}
         disabled={withdrawTxState.loading}
         maxValue={maxAmountToWithdraw.toString(10)}
+        balanceText={<Trans>Supply balance</Trans>}
```

Comment:
Should this be `Available`?

Reply:
In case the amount available to withdraw < supply balance

Reply:
Yes, in all cases? Or just the edge-case when the liquidity isn't enough to cover?

Reply:
Right, I think optimally it makes sense to just state that for that one case, and `Supply balance` for all others